src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -123,34 +123,28 @@ private static void runTabulationTest(String stem, String expectedException) {
       compareJsons(config, stem, timestampString, null);
     }
 
-    // If this is a Dominion tabulation test, also check the converted output file.
-    boolean isDominion =
-        config.rawConfig.cvrFileSources.stream()
-            .anyMatch(source -> ContestConfig.getProvider(source) == Provider.DOMINION);
-    if (isDominion) {
-      String expectedPath =
-          getTestFilePath(
-              stem,
-              "_contest_"
-                  + config.rawConfig.cvrFileSources.get(0).getContestId()
-                  + "_expected.csv");
-      assertTrue(fileCompare(session.getConvertedFilesWritten().get(0), expectedPath));
-    }
-
     cleanOutputFolder(session);
   }
 
-  // helper function to support running convert-to-cdf function
-  private static void runConvertToCdfTest(String stem) {
+  // helper function to support running convert-to-cdf and convert-to-csv functions
```

Comment:
Nit: This implies to me that there's a `convert-to-csv` CLI, which I don't believe is the case.